### ISSUE TITLE
docs: fix stackblitz link on readme

### ISSUE THREAD

**Issue by ayoayco** [ayoayco/astro-reactive-library#156]
This needs to be an HTML image link like the others 😅
![image](https://user-images.githubusercontent.com/4262489/198607649-7c36a0da-cfad-4df3-a3b7-a207c9f1a2b8.jpeg)

**Comment by Anish-Sarawgi** [comment]
Can i work on this?

**Comment by ayoayco** [comment]
Yes @Anish-Sarawgi :) make sure images/badges and links are working on the document. 🚀

**Comment by Anish-Sarawgi** [comment]
Hey @ayoayco,
I have sent the PR. Please review it.

**Comment by ayoayco** [comment]
One change request @Anish-Sarawgi :)

**Comment by Anish-Sarawgi** [comment]
> One change request @Anish-Sarawgi :)

I have made the required changes. Please have a look.
